docs/node/testnet/README.md

```diff
@@ -38,12 +39,9 @@ Feel free to use other seed nodes besides the one provided here.
 :::
 
 * [Oasis Core](https://github.com/oasisprotocol/oasis-core) version:
-  * [22.2.11](https://github.com/oasisprotocol/oasis-core/releases/tag/v22.2.11)
-  * To sync from genesis, you need to start with an earlier version first
-    ([read more][handling network upgrades]):
-    * [22.0.3](https://github.com/oasisprotocol/oasis-core/releases/tag/v22.0.3) (until epoch **15056**)
+  * [23.0](https://github.com/oasisprotocol/oasis-core/releases/tag/v23.0)
 * [Oasis Rosetta Gateway](https://github.com/oasisprotocol/oasis-rosetta-gateway) version:
-  * [2.5.0](https://github.com/oasisprotocol/oasis-rosetta-gateway/releases/tag/v2.5.0)
+  * [2.6.0](https://github.com/oasisprotocol/oasis-rosetta-gateway/releases/tag/v2.6.0)
```

Comment:
Doesn't work yet.

Reply:
That's right, we need to merge https://github.com/oasisprotocol/oasis-rosetta-gateway/pull/475 and make a new release.